node/src/test/scala/io/casperlabs/node/configuration/ConfigurationSpec.scala

```diff
@@ -117,7 +117,8 @@ class ConfigurationSpec
       autoProposeEnabled = false,
       autoProposeCheckInterval = FiniteDuration(1, TimeUnit.SECONDS),
       autoProposeMaxInterval = FiniteDuration(1, TimeUnit.SECONDS),
-      autoProposeMaxCount = 1
+      autoProposeMaxCount = 1,
+      maxBlockSize = 10
```

Comment:
```suggestion
      maxBlockSize = 1
```
It seems to be the pattern to use 1 for everything here.